.eslintrc.js

```diff
@@ -255,6 +252,38 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['**/*.ts', '**/*.md/*.ts', '**/*.md/*.typescript'],
+      excludedFiles: ['examples/angular-example/**/*.ts', 'packages/@uppy/angular/**/*.ts'],
+      parser: '@typescript-eslint/parser',
+      plugins: [
+        '@typescript-eslint',
+      ],
+      extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+      ],
+      rules: {
+        'import/prefer-default-export': 'off',
+        '@typescript-eslint/no-explicit-any': 'off',
+      },
+      parserOptions: {
+        babelOptions: {
+          plugins: [['@babel/plugin-proposal-decorators', { decoratorsBeforeExport : true }]],
```

Comment:
I suppose decorators are only used in angular so if that's ignored do we need this?

Reply:
We need it for blog posts IIRC. Let me put it in a separate override so it's clearer why it's here.